subsys/power/Kconfig

```diff
@@ -4,6 +4,11 @@ menu "OS Power Management"
 
 source "subsys/power/policy/Kconfig"
 
+config PM_CONTROL_STATE_LOCK
+	bool "Enable PM state locking capability"
+	help
+	  Enable OS Power Management state locking capability.
```

Comment:
Can you add some more details about the context of this new capability and what it is good for?